substrate/fd

```diff
@@ -42,21 +44,25 @@ namespace substrate
 		constexpr fd_t() noexcept = default;
 		constexpr fd_t(const int32_t fd_) noexcept : fd{fd_} { }
 		fd_t(const char *const fileName, const int flags, const mode_t mode = 0) noexcept :
-			fd{internal::fdopen(fileName, flags, mode)} { }
+			fd{internal::fdopen(fileName, flags, mode)}
+		{ }
 		fd_t(const std::string &fileName, const int flags, const mode_t mode = 0) noexcept :
-			fd{internal::fdopen(fileName.c_str(), flags, mode)} { }
+			fd{internal::fdopen(fileName.c_str(), flags, mode)}
+		{ }
 #if __cplusplus >= 201703L
 		fd_t(const std::string_view &fileName, const int flags, const mode_t mode = 0) noexcept :
-			fd{internal::fdopen(fileName.data(), flags, mode)} { }
+			fd{internal::fdopen(fileName.data(), flags, mode)}
+		{ }
 #endif
 #if defined(SUBSTRATE_ALLOW_STD_FILESYSTEM_PATH)
 		fd_t(const std::filesystem::path &fileName, const int flags, const mode_t mode = 0) noexcept :
-			fd{internal::fdopen(fileName.c_str(), flags, mode)} { }
+			fd{internal::fdopen(fileName.c_str(), flags, mode)}
+		{ }
 #endif
 		fd_t(fd_t &&other) noexcept : fd_t{}
 			{ *this = std::move(other); }
 
-		~fd_t() noexcept
+		~fd_t() noexcept override
```

Comment:
Could this use `final`instead of `override`? Should allow the compiler to optimise out the vtable penalty if using only fd_t directly